### ISSUE TITLE
Empty workspace randomly flashes with alert.

### ISSUE THREAD

**Issue by scottopell** [workspacer/workspacer#38]
I have a workspace with no windows in it (the text is grey to indicate this) and randomly the indicator will start flashing.

When I go there it dismisses the flashing properly

**Comment by rickbutton** [comment]
I've added some more logging to diagnose this. Will check in next time it happens.

**Comment by rickbutton** [comment]
this is because the flashing event is getting fired when you focus or click on the desktop (progman/explorer).

This should be getting filtered out, but isn't, should be a simple fix.

**Comment by Janthor** [comment]
I just found a fix for it, simply adding the ``WorkerW`` class to the exclusions makes it so that desktop clicks don't trigger a notification alarm. (Can't think of any side-effects to this, since I don't know of a use of the WorkerW class that would require workspacer functionality.
```csx
    context.WindowRouter.AddFilter((window) => !window.Class.Equals("WorkerW")); //Ignores desktop
 ```

**Comment by josteink** [comment]
Closed by #360.
